### PR TITLE
mailmap: Update mailmap with full names and  emails

### PR DESCRIPTION
**Pull request by nashif**
Helps with `git shortlog -ns`

### CODE REVIEW

File: .mailmap [hunk 1]
```diff
@@ -20,3 +20,11 @@ Felipe Neves <ryukokki.felipe@gmail.com> <ryukokki.felipe@gmail.com>
...
 Anas Nashif <anas.nashif@intel.com> <anas.nashif@intel.com>
 Ruud Derwig <Ruud.Derwig@synopsys.com> <Ruud.Derwig@synopsys.com>
+Flavio Arieta Netto <flavio@exati.com.br>
+Nishikant Nayak <nishikantax.nayak@intel.com>
+Justin Watson <jwatson5@gmail.com>
+Johann Fischer <j.fischer@phytec.de>
+Jun Li <jun.r.li@intel.com>
+Xiaorui Hu <xiaorui.hu@linaro.org>
+Yannis Damigos <giannis.damigos@gmail.com> <ydamigos@iccs.gr>
+Vinayak Kariappa Chettimada <vinayak.kariappa.chettimada@nordicsemi.no> <vinayak.kariappa.chettimada@nordicsemi.no> <vich@nordicsemi.no> <vinayak.kariappa@gmail.com>
```
Comment (carlescufi) [inline .mailmap]: duplicate `<vinayak.kariappa.chettimada@nordicsemi.no>`

**Review by carlescufi — CHANGES_REQUESTED** [review]

File: .mailmap (hunk 1 above)
Comment (cvinayak) [reply .mailmap]: remove <vinayak.kariappa@gmail.com>

**Review by cvinayak — CHANGES_REQUESTED** [review]

File: .mailmap (hunk 1 above)
Comment (nashif) [reply .mailmap]: looks like you used it  to commit something in the past, and the point of this change to attribute this to you and not consider it some other person.

**Review by jhedberg — APPROVED** [review]

**Comment by jhedberg** [comment]
> remove vinayak.kariappa@gmail.com

@cvinayak the purpose of this file is to document aliases for different identities the same person has used in the commit history. That way we get correct results e.g. for `git shortlog`. Are you saying this gmail alias is already part of the file or that you don't have any commits in the history that use that address?

File: .mailmap (hunk 1 above)
Comment (nashif) [reply .mailmap]: see this 99ae80b6323f96ae33fc5f500614babf4056b823

**Review by carlescufi — APPROVED** [review]

**Comment by cvinayak** [comment]
@jhedberg I should not have used gmail id. But it appears in git  log, its got in once. Hence, its ok to have it in the mailmap.

**Review by cvinayak — APPROVED** [review]

**Pull request merged**